netdeploy/remote/deployedNetwork.go

```diff
@@ -283,9 +338,424 @@ func (cfg DeployedNetwork) BuildNetworkFromTemplate(buildCfg BuildConfig, rootDi
 		return
 	}
 
+	if cfg.createBoostrappedNetwork {
+		fmt.Println("Generating db files ")
+
+		cfg.GenerateDatabaseFiles(cfg.BootstrappedNet, genesisFolder)
+	}
+
 	return
 }
 
+//GenerateDatabaseFiles generates database files according to the configurations
+func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, genesisFolder string) error {
+
+	fmt.Printf("starts %v\n", time.Now())
+	accounts := make(map[basics.Address]basics.AccountData)
+
+	genesis, err := bookkeeping.LoadGenesisFromFile(genesisFolder + "/genesis.json")
+	if err != nil {
+		return err
+	}
+
+	var bootstrappedNet netState
+
+	bootstrappedNet.genesisID = genesis.ID()
+	bootstrappedNet.genesisHash = crypto.Hash([]byte(genesis.ID()))
+	srcWallet := getGenesisAlloc(fileCfgs.SourceWalletName, genesis.Allocation)
+	if srcWallet.Address == "" {
+		return fmt.Errorf("error finding source wallet address")
+
+	}
+
+	rewardsPool := getGenesisAlloc("RewardsPool", genesis.Allocation)
+	if rewardsPool.Address == "" {
+		return fmt.Errorf("error finding source rewards ppol address")
+
+	}
+
+	feeSink := getGenesisAlloc("FeeSink", genesis.Allocation)
+	if feeSink.Address == "" {
+		return fmt.Errorf("error finding fee sink address")
+
+	}
+	src, err := basics.UnmarshalChecksumAddress(srcWallet.Address)
+
+	poolAddr, err := basics.UnmarshalChecksumAddress(rewardsPool.Address)
+	if err != nil {
+		return err
```

Comment:
you might want to return here a more descriptive error - something that will tell you that the problem is with the reward pool address.